bindings/c/tag_c.h

```diff
@@ -58,7 +58,7 @@ extern "C" {
 
 /*
  * These are used for type provide some type safety to the C API (as opposed to
- * using void *, but pointers to them are simply cast to the corresponding C++
+ * using void * ), but pointers to them are simply cast to the corresponding C++
```

Comment:
What about changing "These are used for type provide some type safety" to "These are used to provide some type safety"? Why a space after "void *"?